lru/__init__.py

```diff
@@ -183,15 +183,15 @@ def cleanup(self):
         next_expire = None
         for k in self.__expire_times:
             if self.__expire_times[k] < t:
-                self.__delete__(k)
+                self.__delitem__(k)
```

Comment:
This PR makes sense, although it will need to be adjusted when/if #13 would get merged in first.